autotest/evaluate/test_api_evaluate_turbomind.py

```diff
@@ -0,0 +1,90 @@
+import pytest
```

Comment:
This file is a copy of `test_api_evaluate_pytorch`.
Could you provide a unified `test_api_evaluation.py`?